data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -1012,12 +1048,16 @@ Sub-Network        |  L2  |  | TSN  |  | UDP  |
         provide these functions along a flow's path through the
         network. See  <xref target="I-D.ietf-detnet-ip"/> and <xref
         target="control-management-requirements"/> for further
-        discussion of these requirements.
+        discussion of these requirements. Some forms of protection
+        may enforce packet loss or change 

```

Comment:
s/ensure/minimize

Reply:
Please clarify what we intend to say here.

Reply:
I believe you mean s/enforce/minimize/